src/RequestGroupMan.h

```diff
@@ -195,6 +198,18 @@ class RequestGroupMan {
 
   bool removeReservedGroup(a2_gid_t gid);
 
+  bool getOptimizeSimultaneousDownloads() const
+  {
+    return optimizeSimultaneousDownloads_;
+  }
+
+  void setOptimizeSimultaneousDownloads(bool flag)
```

Comment:
true it is not used (only the other one, now renamed with 'setup' as per your suggestion is used), i had just put it for completeness of set/get, but i will remove.